README.md

```diff
@@ -25,6 +25,11 @@ Or use script in case you have openSUSE:
 # sudo ./openSUSE_vagrant_setup.sh
 ```
 
+#### libvirtd on a remote machine
```

Comment:
ok, how it works without ssh? Anything needs to be setup before hand?
I go this error if I comment out ssh:
`Error while connecting to libvirt: Error making a connection to libvirt URI qemu:///system?no_verify=1&keyfile=/home/dkondratenko/.ssh/id_rsa:
Call to virConnectOpen failed: authentication unavailable: no polkit agent available to authenticate action 'org.libvirt.unix.manage'`

Reply:
Usually your desktop env (eg. gnome-shell) does provide a polkit-agent (which then asks for a password). Which desktop do you use?
To not need to tell the password every time, add yourself to the `libvirt` group.

Reply:
@denisok do you want me to add a section to the README that you need a polkit-agent or add yourself to the libvirt group?

Reply:
I don't use desktop environment. BTW our CI doesn't either.

Reply:
the question, why we need this change, if it automated with ssh now ?
imagine 10-20 nodes deployment - why to enter password all the time ?

Reply:
No. The solution is to add yourself to the `libvirt` group. This is simpler than installing, starting and enabling sshd and then ssh-copy-id to localhost. And imo it's nicer to not need a local sshd running.
Is the CI setup somewhere automated? Than I can have a look to add the jenkins user (or whatever user is used) to the libvirt group

Reply:
@denisok I can also drop that commit if you think it's not useful. Just let me know.